Form/Type/FileType.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\MediaBundle\Form\Type;
+
+use Symfony\Cmf\Bundle\MediaBundle\File\UploadFileHelperInterface;
+use Symfony\Cmf\Bundle\MediaBundle\Form\DataTransformer\ModelToFileTransformer;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+class FileType extends AbstractType
+{
+    private $dataClass;
+    private $uploadFileHelper;
+
+    /**
+     * @param string                    $class
+     * @param UploadFileHelperInterface $uploadFileHelper
+     */
+    public function __construct($class, UploadFileHelperInterface $uploadFileHelper)
+    {
+        $this->dataClass = $class;
+        $this->uploadFileHelper = $uploadFileHelper;
+    }
+
+    public function getParent()
```

Comment:
please add `/** {@inheritDoc} */` to methods that are inherited and need no further comment